django-backend/fecfiler/transactions/views.py

```diff
@@ -396,6 +438,26 @@ def save_transaction(self, transaction_data, request):
         delete_carried_forward_loans_if_needed(transaction_instance, committee_id)
         delete_carried_forward_debts_if_needed(transaction_instance, committee_id)
 
+        # Handle changing general_election_year for Schedule F transactions
+        if (
```

Comment:
save_transaction is getting pretty big.  I know it would mean passing in more info to ```recalculate_schedule_f_aggregates```, but if we did, could we not figure out ALL of the transactions that need to be recalculated in one place rather than spread across 3 spots in save_transaction?